python/grass/pygrass/modules/interface/testsuite/test_modules.py

```diff
@@ -53,6 +53,44 @@ def test_rsun(self):
         out.close()
 
 
+class TestModulesJsonDictExport(TestCase):
+    def test_rinfo_simple(self):
+        """Test if a Module can be exported to json dict"""
+
+        Module("r.info", map="elevation", run_=False).get_json_dict()
+
+    def test_rinfo_ov(self):
+        """Test if a Module can be exported to json dict with verbose flag"""
+
+        Module("r.info", verbose=True, map="elevation", run_=False).get_json_dict()
+
+    def test_rinfo_ov_export(self):
+        """Test if a Module can be exported to json dict with overwrite
+        and verbose flags and results exported to CSV"""
+
+        Module(
+            "r.info",
+            verbose=True,
+            map="elevation",
+            flags="g",
+            run_=False,
+        ).get_json_dict(export="CSV")
+
+    def test_rslopeaspect_ov_export(self):
+        """Test if a Module can be exported to json dict with overwrite
+        and verbose flags and results exported to CSV"""
+
+        Module(
+            "r.slope.aspect",
+            elevation="elevation",
+            slope="slope",
+            aspect="aspect",
+            overwrite=True,
+            verbose=True,
+            run_=False,
+        ).get_json_dict(export="GTiff")
```

Comment:
Nothing against smoke tests in general, but these tests are unnecessary simple. Please add assertions to move it beyond smoke tests. It does not really tell what is the expected output.